types/css-generator/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for css-generator 1.0
+// Project: https://github.com/luizbills/css-generator.js
+// Definitions by: Septs <https://github.com/septs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.0
+
+export interface Options {
+    indentation?: string;
+}
+
+export class Generator {
+    private constructor();
```

Comment:
The constructor has one optional parameter `options`
https://github.com/luizbills/css-generator.js/blob/master/src/generator.js#L3

Reply:
the class un-export

Reply:
Yes. I know but the type should be there.